### PR TITLE
docs(profile readme): bring more love in Tractus-X

### PR DESCRIPTION
**Pull request by stephanbcbauer**
## Description

We discussed, today, adding more content to the profile readme. This is just a first DRAFT. Please have a look. THX

How would it look like right now:

image
image
image
image
image
image

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [X] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [X] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,8 +1,77 @@
...
+![Tractus-X Logo](https://avatars.githubusercontent.com/u/102587798?s=200&v=4)
 
-For the latest Tractus-X release, see the [Changelog](https://github.com/eclipse-tractusx/tractus-x-release/blob/main/CHANGELOG.md) of our release repository [tractus-x-release](https://github.com/eclipse-tractusx/tractus-x-release).
+## About Eclipse Tractus-X
 
-Please follow the [Security Issue Reporting Guidelines](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-01#security-file) if you come across any security vulnerabilities or concerns.
+Eclipse Tractus-X is a collaborative, open-source initiative aimed at driving the digital transformation of the automotive industry and beyond. Our mission is to enable secure, efficient data exchange and seamless
+collaboration across the supply chain using trusted open standards.
+
+Tractus-X is the home of the refernce implementations used in Catena-X. Tractus-X develops solutions that foster interoperability between various stakeholders, providing an open ecosystem for data-driven applications.
```
Comment (jSchuetz88) [inline profile/README.md]:
```suggestion
Tractus-X is the home of the reference implementations used in Catena-X. Tractus-X develops solutions that foster interoperability between various stakeholders, providing an open ecosystem for data-driven applications.
```

Comment (jSchuetz88) [reply profile/README.md]: Should Catena-X be explicitly mentioned here? Or perhaps formulate it more openly so not to restrict it to Catena-X as Catena-X is (potentially) just the first, but other initiatives may join?

Comment (HFocken) [reply profile/README.md]: Should be mentioned - transparency is key - no need to hide - we just need to keep it open and inviting for others and i think Stephans current wording reflects that.

File: profile/README.md [hunk 2]
```diff
@@ -1,8 +1,77 @@
...
+This includes reference implementations and models for **supply chain**, **compliance**, **sustainability**, and **more**.
+
+For the latest Tractus-X release, see the [Changelog](https://eclipse-tractusx.github.io/CHANGELOG) on our [webpage](https://eclipse-tractusx.github.io/).
+
+## Our Vision
+
+Our goal is to create an open, scalable, and secure digital ecosystem (refernce implementations) for the automotive industry that:
```
Comment (HFocken) [inline profile/README.md]: That however is too much catena-x I'd say - tractus-x does not create the ecosystem but supports it by providing a development environment and KITs.

File: profile/README.md [hunk 3]
```diff
@@ -1,8 +1,77 @@
...
+- Enables collaboration between suppliers, manufacturers, and partners.
+- Facilitates transparency and sustainability in automotive industrial processes.
+- Drives innovation across industries through open and community-driven standards.
+- Can easily be adapted/extended to other industries.
+- Ensures data sovereignty and privacy for all participants.
+- Offers an easy entry to various use cases through our [KITs (Keep It Together)](https://eclipse-tractusx.github.io/Kits) approach.
+
+## Key Projects
```
Comment (HFocken) [inline profile/README.md]: Related projects instead of key projects? Tractus-X is a project itself...

File: profile/README.md [hunk 4]
```diff
@@ -1,8 +1,77 @@
...
+
+- **[Eclipse Dataspace Components](https://projects.eclipse.org/projects/technology.edc)**: A solution for secure and standardized data sharing between partners.
+- What else?
```
Comment (HFocken) [inline profile/README.md]: Deleted before merging

File: profile/README.md [hunk 5]
```diff
@@ -1,8 +1,77 @@
...
+
+## How to Get Involved
+
+We are an open-source community, and we welcome contributors from all backgrounds. You can get involved in many ways:
+- **Join our discussions**: Check out the issues in our repositories to see current challenges and join the conversation.
+- **Contribute**: Whether it's code, documentation, or ideas, contributions are always welcome! Start by reading our [Contributing Guide](https://eclipse-tractusx.github.io/docs/oss/how-to-contribute).
+- **Attend our events**: Participate in our [regular](https://eclipse-tractusx.github.io/community/open-meetings#regular-meetings) and [one-time](https://eclipse-tractusx.github.io/community/open-meetings#one-time-meetings).
+- **Collaborate**: Partner with us to help shape the future of digital transformation in the automotive and manufacturing industries.
```
Comment (HFocken) [inline profile/README.md:37]: Can we leave a contact address of a project lead?

File: profile/README.md (hunk 4 above)
Comment (HFocken) [inline profile/README.md]: I'd list Catena-X here

**Review by HFocken — CHANGES_REQUESTED** [review]
Great stuff, thanks Stephan!!

File: profile/README.md (hunk 3 above)
Comment (stephanbcbauer) [reply profile/README.md]: Adapted the headline and also added suggestion for Catena-X Ecosystem and the operating company. .... EDC still OK? What else?

File: profile/README.md (hunk 4 above)
Comment (stephanbcbauer) [reply profile/README.md]: Yes, is just a reminder ;)

File: profile/README.md (hunk 2 above)
Comment (stephanbcbauer) [reply profile/README.md]: It's never too much Catena-X ;)

File: profile/README.md (hunk 5 above)
Comment (stephanbcbauer) [reply profile/README.md:37]: @eclipse-tractusx/automotive-tractusx-project-leads what do you think?

File: profile/README.md (hunk 4 above)
Comment (stephanbcbauer) [reply profile/README.md]: Can you please check the wording and links again?

File: profile/README.md (hunk 2 above)
Comment (stephanbcbauer) [reply profile/README.md]: Is this comment related to the bullet points? Or the first sentence in the Vision?

File: profile/README.md (hunk 4 above)
Comment (stephanbcbauer) [reply profile/README.md]: done

File: profile/README.md (hunk 2 above)
Comment (HFocken) [reply profile/README.md]: The first sentence in the vision

**Comment by matbmoser** [comment]
@stephanbcbauer I would like to add something that we are enabling the base for -X Dataspaces or something like this. Or that we are a community that is committed on bringing Catena-X and Gaia-X dataspaces to live. I don't know how to express it so it sounds good. I have saw the comment @HFocken and I agree with that, but I feel that something is missing but just can't phrase it.


But that is exactly what I meant with "giving more love" to our main readme! Thank you very much for the preparation ;)

**Comment by stephanbcbauer** [comment]
> @stephanbcbauer I would like to add something that we are enabling the base for -X Dataspaces or something like this. Or that we are a community that is committed on bringing Catena-X and Gaia-X dataspaces to live. I don't know how to express it so it sounds good. I have saw the comment @HFocken and I agree with that, but I feel that something is missing but just can't phrase it.
> 
> But that is exactly what I meant with "giving more love" to our main readme! Thank you very much for the preparation ;)

Suggest something :) We can also brainstorm a bit. But i think the related project should somehow connect to the other data spaces?

File: profile/README.md [hunk 6]
```diff
@@ -1,8 +1,79 @@
...
+collaboration across the supply chain using trusted open standards.
+
+Tractus-X is the home of the reference implementations used in Catena-X. Tractus-X develops solutions that foster interoperability between various stakeholders, providing an open ecosystem for data-driven applications.
+This includes reference implementations and models for **supply chain**, **compliance**, **sustainability**, and **more**.
+
+For the latest Tractus-X release, see the [Changelog](https://eclipse-tractusx.github.io/CHANGELOG) on our [webpage](https://eclipse-tractusx.github.io/).
+
+## Our Vision
+
+Our goal is to create an open, scalable, and secure digital ecosystem (refernce implementations) for the automotive industry that:
```
Comment (matbmoser) [inline profile/README.md]: For the automotive and many other industries?

File: profile/README.md [hunk 7]
```diff
@@ -1,8 +1,79 @@
...
+
+- Enables collaboration between suppliers, manufacturers, and partners.
+- Facilitates transparency and sustainability in automotive industrial processes.
+- Drives innovation across industries through open and community-driven standards.
+- Can easily be adapted/extended to other industries.
```
Comment (matbmoser) [inline profile/README.md]: Maybe this is already mentioned here

File: profile/README.md [hunk 8]
```diff
@@ -1,8 +1,79 @@
...
+
+- Join our mailing list: [subscribe here](https://accounts.eclipse.org/mailing-list/tractusx-dev)
+- Follow us on our [webpage](https://eclipse-tractusx.github.io/) and our [news sectcion](https://eclipse-tractusx.github.io/blog)
+- Join our matrix chats at: [Tractus-X](https://matrix.to/#/#automotive.tractusx:matrix.eclipse.org)
+
+## Our Community
+
+Meet the project leads of Eclipse Tractus-X:
+
+| ![Evelyn Gurschler](https://images.weserv.nl/?url=avatars.githubusercontent.com/stephanbcbauer?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Evelyn Gurschler](https://images.weserv.nl/?url=avatars.githubusercontent.com/matbmoser?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Evelyn Gurschler](https://images.weserv.nl/?url=avatars.githubusercontent.com/evegufy?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Daniel Miehle](https://images.weserv.nl/?url=avatars.githubusercontent.com/danielmiehle?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Björn Roy](https://images.weserv.nl/?url=avatars.githubusercontent.com/giterrific?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d)  |
```
Comment (matbmoser) [inline profile/README.md]: Evelyn is mentioned three times in the "alt" option. Maybe we should add our names when specified

File: profile/README.md [hunk 9]
```diff
@@ -1,8 +1,79 @@
...
+|                                     **[Stephan Bauer](https://github.com/stephanbcbauer)**                                      | **[Mathias Brunkow Moser](https://github.com/matbmoser)**           |                                                 **[Evelyn Gurschler](https://github.com/evegufy)**                                                 |                                                **[Daniel Miehle](https://github.com/danielmiehle)**                                                 |                                               **[Björn Roy](https://github.com/giterrific)**                                               |
+
+Meet the [contributors](https://projects.eclipse.org/projects/automotive.tractusx/who) and [committer](https://projects.eclipse.org/projects/automotive.tractusx/who) of Eclipse Tractus-X.
```
Comment (matbmoser) [inline profile/README.md]: Should not the committers come first?

**Review by matbmoser — COMMENTED** [review]
Maybe I would like to add some statistics? @stephanbcbauer do you know any git stats service?

File: profile/README.md (hunk 8 above)
Comment (stephanbcbauer) [reply profile/README.md]: copy pate :) haha

File: profile/README.md (hunk 9 above)
Comment (stephanbcbauer) [reply profile/README.md]: You mean before the contributors? Or Contributor and Committer before Project lead?

File: profile/README.md (hunk 7 above)
Comment (stephanbcbauer) [reply profile/README.md]: yes already mentioned in the vision, first sentence... @HFocken fine with the first sentence now?

File: profile/README.md (hunk 6 above)
Comment (stephanbcbauer) [reply profile/README.md]: done

File: profile/README.md (hunk 8 above)
Comment (stephanbcbauer) [reply profile/README.md]: done

File: profile/README.md (hunk 9 above)
Comment (stephanbcbauer) [reply profile/README.md]: done

**Comment by matbmoser** [comment]
I have found some cool statistic website and have organized a statistics section, which looks like this: 
image
image
image
image


@stephanbcbauer will open an IP check if we can use this platform referencing in the readme...
If not approved I remove this section. Until then what do you think?

**Comment by matbmoser** [comment]
image
@danielmiehle or @maximilianong dont we have a Tractus-X Logo (with tractus-x written, that we can put there? Maybe even with the famous background?

Like this but in white:

Tractus_X_Logo_RGB_Dark_Version_for_Website

I also have a "tractus-x" kit:

Tractus-X KIT v2

**Review by matbmoser — CHANGES_REQUESTED** [review]
Not approving until we have the IP clearance, I will make it also a draft

**Comment by stephanbcbauer** [comment]
> I have found some cool statistic website and have organized a statistics section, which looks like this: image image image image
> 
> @stephanbcbauer will open an IP check if we can use this platform referencing in the readme... If not approved I remove this section. Until then what do you think?

@matbmoser created the Ticket https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/16632

File: profile/README.md [hunk 10]
```diff
@@ -1,8 +1,304 @@
...
+![Tractus-X Logo](https://avatars.githubusercontent.com/u/102587798?s=200&v=4)
 
-For the latest Tractus-X release, see the [Changelog](https://github.com/eclipse-tractusx/tractus-x-release/blob/main/CHANGELOG.md) of our release repository [tractus-x-release](https://github.com/eclipse-tractusx/tractus-x-release).
+## About Eclipse Tractus-X
 
-Please follow the [Security Issue Reporting Guidelines](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-01#security-file) if you come across any security vulnerabilities or concerns.
+Eclipse Tractus-X is a collaborative, open-source initiative aimed at driving the digital transformation of the automotive industry and beyond. Our mission is to enable secure, efficient data exchange and seamless
+collaboration across the supply chain using trusted open standards.
+
+Tractus-X is the home of the reference implementations used in Catena-X. Tractus-X develops solutions that foster interoperability between various stakeholders, providing an open ecosystem for data-driven applications.
```
Comment (evegufy) [inline profile/README.md]:
```suggestion
Eclipse Tractus-X is the home of the reference implementations used in [Catena-X](https://catena-x.net). The project develops solutions that foster interoperability between various stakeholders, providing an open ecosystem for data-driven applications.
```

File: profile/README.md [hunk 11]
```diff
@@ -1,8 +1,304 @@
...
+This includes reference implementations and models for **supply chain**, **compliance**, **sustainability**, and **more**.
```
Comment (evegufy) [inline profile/README.md]:
```suggestion
This includes reference implementations and models for **supply chain**, **compliance**, **sustainability** and **more**.
```

File: profile/README.md [hunk 12]
```diff
@@ -1,8 +1,304 @@
...
+- Follow us on our [webpage](https://eclipse-tractusx.github.io/) and our [news sectcion](https://eclipse-tractusx.github.io/blog)
+- Join our matrix chats at: [Tractus-X](https://matrix.to/#/#automotive.tractusx:matrix.eclipse.org)
+
+## Our Community
+
+Meet the [contributors](https://projects.eclipse.org/projects/automotive.tractusx/who) and [committer](https://projects.eclipse.org/projects/automotive.tractusx/who) and our project leads of Eclipse Tractus-X:
+
+| ![Stephan Bauer](https://images.weserv.nl/?url=avatars.githubusercontent.com/stephanbcbauer?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Mathias Brunkow Moser](https://images.weserv.nl/?url=avatars.githubusercontent.com/matbmoser?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Evelyn Gurschler](https://images.weserv.nl/?url=avatars.githubusercontent.com/evegufy?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Daniel Miehle](https://images.weserv.nl/?url=avatars.githubusercontent.com/danielmiehle?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Björn Roy](https://images.weserv.nl/?url=avatars.githubusercontent.com/giterrific?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d)  |
```
Comment (evegufy) [inline profile/README.md]: I wouldn't single out the project leads here so much, as our community is driven by committers, contributors and project leads. In regards to community, I'd find links to our Matrix chat and dev mailing list more meaningful.

File: profile/README.md [hunk 13]
```diff
@@ -1,8 +1,304 @@
...
+
+## Stay Connected
+
+- Join our mailing list: [subscribe here](https://accounts.eclipse.org/mailing-list/tractusx-dev)
+- Follow us on our [webpage](https://eclipse-tractusx.github.io/) and our [news sectcion](https://eclipse-tractusx.github.io/blog)
+- Join our matrix chats at: [Tractus-X](https://matrix.to/#/#automotive.tractusx:matrix.eclipse.org)
+
+## Our Community
+
+Meet the [contributors](https://projects.eclipse.org/projects/automotive.tractusx/who) and [committer](https://projects.eclipse.org/projects/automotive.tractusx/who) and our project leads of Eclipse Tractus-X:
```
Comment (evegufy) [inline profile/README.md]:
```suggestion
Meet the [contributors](https://projects.eclipse.org/projects/automotive.tractusx/who) and [committer](https://projects.eclipse.org/projects/automotive.tractusx/who) and our [project leads](https://projects.eclipse.org/projects/automotive.tractusx/who) of Eclipse Tractus-X:
```

File: profile/README.md [hunk 14]
```diff
@@ -1,8 +1,304 @@
...
+
+## How to Get Involved
+
+We are an open-source community, and we welcome contributors from all backgrounds. You can get involved in many ways:
+
+- **Join our discussions**: Check out the issues in our repositories to see current challenges and join the conversation.
+- **Contribute**: Whether it's code, documentation, or ideas, contributions are always welcome! Start by reading our [Contributing Guide](https://eclipse-tractusx.github.io/docs/oss/how-to-contribute).
+- **Attend our events**: Participate in our [regular](https://eclipse-tractusx.github.io/community/open-meetings#regular-meetings) and [one-time](https://eclipse-tractusx.github.io/community/open-meetings#one-time-meetings).
+- **Collaborate**: Partner with us to help shape the future of digital transformation in the automotive and manufacturing industries.
+- **Repositories**: Check out our [GitHub repositories](https://github.com/eclipse-tractusx) to see our latest projects and contributions.
```
Comment (evegufy) [inline profile/README.md]:
```suggestion
- **Repositories**: Check out our [GitHub repositories](https://github.com/orgs/eclipse-tractusx/repositories) to see our latest projects and contributions.
```

File: profile/README.md [hunk 15]
```diff
@@ -1,8 +1,304 @@
...
+
+## Our Vision
+
+Our goal is to create an open, scalable, and secure digital ecosystem (refernce implementations) for the automotive and many other industries that:
+
+- Enables collaboration between suppliers, manufacturers, and partners.
+- Facilitates transparency and sustainability in automotive industrial processes.
+- Drives innovation across industries through open and community-driven standards.
+- Ensures data sovereignty and privacy for all participants.
+- Offers an easy entry to various use cases through our [KITs (Keep It Together)](https://eclipse-tractusx.github.io/Kits) approach.
```
Comment (evegufy) [inline profile/README.md]:
```suggestion
Our goal is to create open, scalable and secure reference implementations for a digital ecosystem in the automotive and many other industries that:

- Enable collaboration between suppliers, manufacturers, and partners.
- Facilitate transparency and sustainability in automotive industrial processes.
- Drive innovation across industries through open and community-driven standards.
- Ensure data sovereignty and privacy for all participants.
- Offer an easy entry to various use cases through our [KITs (Keep It Together)](https://eclipse-tractusx.github.io/Kits) approach.
```

File: profile/README.md [hunk 16]
```diff
@@ -1,8 +1,304 @@
...
-- [Eclipse Tractus-X Project page](https://eclipse-tractusx.github.io/)
-- [Eclipse Foundation Tractus-X Project page](https://projects.eclipse.org/projects/automotive.tractusx)
+![Tractus-X Logo](https://avatars.githubusercontent.com/u/102587798?s=200&v=4)
 
-For the latest Tractus-X release, see the [Changelog](https://github.com/eclipse-tractusx/tractus-x-release/blob/main/CHANGELOG.md) of our release repository [tractus-x-release](https://github.com/eclipse-tractusx/tractus-x-release).
+## About Eclipse Tractus-X
 
-Please follow the [Security Issue Reporting Guidelines](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-01#security-file) if you come across any security vulnerabilities or concerns.
+Eclipse Tractus-X is a collaborative, open-source initiative aimed at driving the digital transformation of the automotive industry and beyond. Our mission is to enable secure, efficient data exchange and seamless
+collaboration across the supply chain using trusted open standards.
```
Comment (evegufy) [inline profile/README.md]:
```suggestion
Eclipse Tractus-X is a collaborative, open-source project aimed at driving the digital transformation of the automotive industry and beyond. Our mission is to enable secure, efficient data exchange and seamless collaboration across the supply chain using trusted open standards.
```

**Review by evegufy — CHANGES_REQUESTED** [review]

**Comment by stephanbcbauer** [comment]
@matbmoser , @evegufy answer provided by eclipse -> https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/16632#note_2837759

File: profile/README.md (hunk 14 above)
Comment (matbmoser) [reply profile/README.md]: nice point

File: profile/README.md (hunk 12 above)
Comment (matbmoser) [reply profile/README.md]: Its cool so see some faces in the main readme file. If I am honest I would let us there so that people can contact us.

Comment (stephanbcbauer) [reply profile/README.md]: @evegufy, @matbmoser  I updated the description and added the links to mailinglist and matrix chat :) -> The project lead faces are still there :) Do you really want to delete them? :)

**Comment by stephanbcbauer** [comment]
> @matbmoser , @evegufy answer provided by eclipse -> https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/16632#note_2837759

@matbmoser can you please add some suggestions of useful statistics? I think 3 should be enough? Thx

**Comment by matbmoser** [comment]
Which of the ones I proposed I should keep? @stephanbcbauer @evegufy @HFocken ??

File: profile/README.md (hunk 12 above)
Comment (evegufy) [reply profile/README.md]: @stephanbcbauer I'd really not treat project leads any different, so, yes, I'd go for deleting them, sorry if that's not the anticipated answer :)

**Comment by evegufy** [comment]
> Which of the ones I proposed I should keep? @stephanbcbauer @evegufy @HFocken ??

If I'd had to choose 3, then: Organization Overview, Where Are Our GitHub Participants Based? and Engagement

**Comment by stephanbcbauer** [comment]
> > Which of the ones I proposed I should keep? @stephanbcbauer @evegufy @HFocken ??
> 
> If I'd had to choose 3, then: Organization Overview, Where Are Our GitHub Participants Based? and Engagement

fine for me :) Happy to see this online in the near future :)

File: profile/README.md (hunk 12 above)
Comment (stephanbcbauer) [reply profile/README.md]: OK :) will delete us/them :)

**Comment by matbmoser** [comment]
nOW IS THIS TEHRE

image

**Comment by stephanbcbauer** [comment]
@matbmoser thx. ready for review? ;) Did we miss the TRG Link?

**Comment by matbmoser** [comment]
Now its done!

**Review by matbmoser — APPROVED** [review]

**Pull request merged**